login-fire-form.html

```diff
@@ -298,6 +298,15 @@
         type: String
       },
 
+      /**
+       * If sign up suggestion should be visible.
+       *
+       * @type {Boolean}
+       */
+      noSignUp: {
```

Comment:
The property is already defined inside the `login-fire-form-behavior`.

Reply:
Hmm, I cannot see it... Do you want me to put it into the behavior?

Reply:
I was in the wrong behavior ;)